### PR TITLE
S2U-12 5.1.1.8 Improving i18n and feedback on calculated questions

### PR DESCRIPTION
**Pull request by jesusmmp**
Some fixes.
https://sakaiproject.atlassian.net/browse/S2U-12

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties [hunk 1]
```diff
@@ -178,6 +178,8 @@ samigo_formula_error_5=5\: Parte inesperada
 samigo_formula_error_6=6\: Final inesperado de la expresi\u00f3n 
 samigo_formula_error_7=7\: Valor esperado
 samigo_formula_error_9=9\: Variable no definida en las instrucciones
+samigo_formula_error_10=10: Variable no definida en el comentario a la respuesta correcta
+samigo_formula_error_11=11: Variable no definida en el comentario a la respuesta incorrecta
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:182]: Could you add these strings in the _ca and _eu files?

Comment (jesusmmp) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:182]: Done!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties [hunk 2]
```diff
@@ -178,6 +178,8 @@ samigo_formula_error_5=5\: Parte inesperada
 samigo_formula_error_6=6\: Final inesperado de la expresi\u00f3n 
 samigo_formula_error_7=7\: Valor esperado
 samigo_formula_error_9=9\: Variable no definida en las instrucciones
+samigo_formula_error_10=10: Variable no definida en el comentario a la respuesta correcta
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:181]: @pablosanroman can you help here whenever you can? Thanks

**Review by mpellicer — APPROVED** [review]

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties (hunk 2 above)
Comment (jesusmmp) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:181]: @mpellicer All the translations are on the PR. This is ready to merge! :)

Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:181]: @mpellicer 
+ samigo_formula_error_10=10:  Aldagaia ez dago definituta feedbackean (ongi dagoen erantzunarena)

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_es.properties:181]: Thanks @PabloSanRoman, I've included a fix for this.

**Pull request merged**